lib/CModHandler.h

```diff
@@ -313,6 +318,21 @@ class DLL_LINKAGE CModHandler
 			{
 				BLACK_MARKET_MONTHLY_ARTIFACTS_CHANGE = true;
 			}
+			
+			if(version >= 782)

```

Comment:
Wrong version, yours is 788

Reply:
Version problem git fixed in merge commit

Reply:
But it looks like its not.